src/SimpleExcelReader.php

```diff
@@ -233,7 +233,7 @@ public function headerRowFormatter(callable $callback)
 
     protected function trim(string $header): string
     {
-        return call_user_func_array('trim', array_filter([$header, $this->trimHeaderCharacters]));
+        return call_user_func_array('trim', array_merge([$header], array_filter([$this->trimHeaderCharacters])));
```

Comment:
This statement is becoming unreadable to me. Could you split this up in multiple smaller statements?

Reply:
Yeah updated. Tried my best.